File: Modulo 2/Clase03/app/src/main/java/com/area51/clase03/AndroidApplication.java
package com.area51.clase03;

import android.app.Application;

import io.realm.Realm;
import io.realm.RealmConfiguration;

public class AndroidApplication extends Application {
    @Override
    public void onCreate() {
        super.onCreate();

        Realm.init(this);
        RealmConfiguration configuration =
                new RealmConfiguration.Builder()
                        .name("clase03")
                        .schemaVersion(1)
                        .build();
        Realm.setDefaultConfiguration(configuration);
    }
}

File: Modulo 3/ClaseMVP/app/src/main/java/com/area51/clasemvp/AndroidApplication.kt
package com.area51.clasemvp

import android.app.Application
import io.realm.Realm
import io.realm.RealmConfiguration

open class AndroidApplication : Application() {
    override fun onCreate() {
        super.onCreate()

        Realm.init(this)
        val configuration = RealmConfiguration.Builder()
            .schemaVersion(1)
            .name("clase_mvp.realm")
            .build()
        Realm.setDefaultConfiguration(configuration)
    }
}